src/components/message-input/index.test.tsx

```diff
@@ -96,6 +100,18 @@ describe('MessageInput', () => {
     expect(dropzone.find(ReplyCard).exists()).toBe(true);
   });
 
+  it('renders MessageAudioRecorder', function () {
+    const wrapper = subject({});
+    const dropzone = wrapper.find(Dropzone).shallow();
+
+    expect(dropzone.find(MessageAudioRecorder).exists()).toBe(false);
+
+    (wrapper.instance() as any).startMic();
```

Comment:
```suggestion
    wrapper.find('.record-voice__icon').simulate('click')
```
In most of our other tests we're avoiding calling instance. Can we simulate the user event for click?